common/src/main/java/com/gregtechceu/gtceu/api/registry/registrate/MachineBuilder.java

```diff
@@ -91,8 +93,9 @@ public class MachineBuilder<DEFINITION extends MachineDefinition> extends Builde
     private Consumer<ItemBuilder<? extends MetaMachineItem, ?>> itemBuilder;
     @Setter
     private NonNullConsumer<BlockEntityType<BlockEntity>> onBlockEntityRegister = MetaMachineBlockEntity::onBlockEntityRegister;
-    @Setter
-    private GTRecipeType recipeType;
+    private GTRecipeType[] recipeTypes;
+    @Persisted
```

Comment:
persited is unnecessary

Reply:
no it isn't that's the active type, which does need to be saved

Reply:
?? be saved in the `MachineBuilder`??

Reply:
oh damn i didn't look.